content/projects/naomi.md

```diff
@@ -0,0 +1,18 @@
++++
+date = "2020-02-03"
+title = "Naomi"
++++
+
+[https://naomi.unaids.org](https://naomi.unaids.org/)
+
+A web interface for a model estimating various HIV epidemic indicators at a regional level. For use by 
```

Comment:
We should link to Jeff somewhere here if he has a web presence. I think the poster would be better displayed as a thumbnail linking through to a pdf?